Sources/App/Model/Speaker.swift

```diff
@@ -2,15 +2,60 @@
 //  Speaker.swift
 //  
 //
-//  Created by Alex Logan on 19/03/2022.
+//  Created by Joe Williams on 09/04/2022.
```

Comment:
😢

Reply:
haha I don't know how this happened, maybe I recreated it! sorry!!